promtail-config.yaml

```diff
@@ -0,0 +1,50 @@
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
+
+clients:
+  - url: http://172.17.0.3:3100/loki/api/v1/push
```

Comment:
@tSudharshan , @naveen-armedia , is it OK to hardcode an IP address in this URL?

Reply:
as far as i know it won't work, it should be override with loki container ipaddress